pandas/tests/frame/test_timeseries.py

```diff
@@ -706,6 +723,44 @@ def test_between_time_raises(self):
         with pytest.raises(TypeError):  # index is not a DatetimeIndex
             df.between_time(start_time='00:00', end_time='12:00')
 
+    @pytest.mark.parametrize('axis', [
```

Comment:
also test for 0, 1

Reply:
the new axis fixture will handle this, can you use that

Reply:
which new axis fixture?

Reply:
rebase in master and axis is available

Reply:
why is this not just using the axis fixture?

Reply:
I couldn't figure out what the axis fixture is. Is there documentation about this?

Reply:
Here's the location of it. It just supplies the various arguments that are typically valid for an `axis` parameter

https://github.com/pandas-dev/pandas/blob/c04b7739b7caa04496c876b19c37e0decf88af78/pandas/conftest.py#L65